requirements.txt

```diff
@@ -1,6 +1,25 @@
 aiohttp==3.7.4
-aiohttp_jinja2==1.1.0
+aiohttp-jinja2==1.1.0
```

Comment:
just out of curiosity, why so many new requirements? :) And do we need to set a version for each?

Reply:
Apart from the original `requirements.txt`, I only installed `pyppeteer`. Maybe these are dependencies? I think setting a version would make Snare more stable.